modules/rosa-upgrading-preparing-4-8-to-4-9.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * upgrading/rosa-upgrading-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-upgrading-requirements-49_{context}"]
+= Requirements for upgrading to OpenShift 4.9
+
+You must meet the following requirements before upgrading a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS) from version 4.8 to 4.9:
+
+* Update the AWS Identity and Access Management (IAM) account-wide roles and policies, including the Operator policies to version 4.9.
```

Comment:
Would this be a two-step procedure since they're done in succession? I wonder if having the numbered might also break up the monotony of the bullet points (of which there are a lot).